storybookjs/aem
# Feature/prepare script testing

Issue: Prepare script failing. Npm install failing

## What I did
I updated the prepare script to include an input, and allow js. I also added an empty preview.ts file so the prepare script doesn't fail because the lack of the script.

## How to test
Run `yarn prepare` from app/aem folder. See that it doesn't fail. Then test the aem-kitchen sink. See it doesn't fail. And doesn't throw the error that Runtime is not a constructor. 

- Is this testable with Jest or Chromatic screenshots? No
- Does this need a new example in the kitchen sink apps? No 
- Does this need an update to the documentation? No
